src/common/components/CardActionButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Button, CardActions, Link } from '@mui/material';
+
+const CardActionButton = ({ label, destination }) => {
```

Comment:
Issue: This name is still not describing the intent of the component, I think `CardActionRouterLink` is better. It seems in here you are using the `Link` component from `@mui/material` and the expectation from the places where you replace this is to use `import { Link } from 'react-router-dom';`. Is this still working as expected?